test/test_extended_models.py

```diff
@@ -59,6 +60,24 @@ def test_get_model_weights(name, weight):
     assert models.get_model_weights(name) == weight
 
 
+@pytest.mark.parametrize("copy_fn", [copy.copy, copy.deepcopy])
+@pytest.mark.parametrize(
+    "name",
+    [
+        "resnet50",
+        "retinanet_resnet50_fpn_v2",
+        "raft_large",
+        "quantized_resnet50",
+        "lraspp_mobilenet_v3_large",
+        "mvit_v1_b",
+    ],
+)
+def test_weights_copyable(copy_fn, name):
+    weights = models.get_model_weights(name)
+    copied_weights = copy_fn(weights)
```

Comment:
Should we be copying all the weights/enum or a single instance of them?

Reply:
You are right, it should be a single member rather then the whole enum.